app/Repositories/Eloquents/NotificationEloquentRepository.php

```diff
@@ -90,11 +88,23 @@ public function getNotifications($with = [], $data = [], $dataSelect = ['*'])
 
                 case config('model.target_type.vote'):
                     $book = $record->target->review->book;
-                    $record = array_add($record, 'message', config('view.notifications.vote') . $book->title);
+                    if ($record->target->status == 1) {
+                        $record = array_add(
+                            $record, 
+                            'message', 
+                            config('view.notifications.upvote') . $book->title
+                        );
+                    } elseif ($record->target->status == -1) {
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.SuperfluousWhitespace.EndLine</br>**Severity**: error</br>**File**: app/Repositories/Eloquents/NotificationEloquentRepository.php</br>**Line**: 99</br>Whitespace found at end of line